scripts/dump_html.py

```diff
@@ -8,16 +8,41 @@
 import os
 import sys
 
-with open(sys.argv[1]) as f:
-    chars = f.read()
 
-print("// dump from file: %s" %os.path.basename(sys.argv[1]))
+def dump(
+    file_from: str = sys.argv[1],
+    file_to: str | None = None,
+    var_name: str = sys.argv[2],
+):
```

Comment:
I would prefer seeing these parameters inside of `if __name__...` instead of defaults. But that's just my personal taste.

Reply:
@danielkucera I have updated the function declaration but I still left  `file_to`, as it is more of an optional argument more than the default value